solr/conf/schema.xml

```diff
@@ -317,6 +317,7 @@
    <dynamicField name="*_isim"  type="pint"      stored="true"  indexed="true"  multiValued="true"  />
    <dynamicField name="*_is"    type="pint"      stored="true"  indexed="false" multiValued="false" />
    <dynamicField name="*_ii"    type="pint"      stored="false" indexed="true"  multiValued="false" />
```

Comment:
Should we remove all the non-stored dynamic fields?  It seems like leaving these in could get people into a situation where they can't do atomic-updates.

Reply:
🤷‍♂️ we use some of them in copy fields, and I figured it might be nice for downstream not to throw them away immediately.